.gitignore

```diff
@@ -8,3 +8,4 @@ tcping.exe
 *.code-workspace
 patches/
 *.db
```

Comment:
ignore .tool-versions is useful for folks like me that use asdf :)
I hope this is okay?